crates/astria-sequencer-client/src/extension_trait.rs

```diff
@@ -436,6 +460,41 @@ pub trait SequencerClientExt: Client {
         self.get_balance(address, 0u32).await
     }
 
+    /// Returns the allowed fee assets at a given height.
+    ///
+    /// # Errors
+    ///
+    /// - If calling tendermint `abci_query` RPC fails.
+    /// - If the bytes contained in the abci query response cannot be deserialized as an
+    ///  `astria.protocol.asset.v1alpha1.AllowedFeeAssetIdsResponse`.
+    /// - If the raw response cannot be converted to the native type.
+    async fn get_allowed_fee_asset_ids(&self) -> Result<AllowedFeeAssetIdsResponse, Error> {
+        let path = "asset/allowed_fee_asset_ids".to_string();
+
+        let response = self
+            .abci_query(Some(path), vec![], Some(0u32.into()), false)
+            .await
+            .map_err(|e| Error::tendermint_rpc("abci_query", e))?;
+
+        let proto_response =
+            astria_core::generated::protocol::asset::v1alpha1::AllowedFeeAssetIdsResponse::decode(
+                &*response.value,
+            )
+            .map_err(|e| {
+                Error::abci_query_deserialization(
+                    "astria.protocol.asset.v1alpha1.AllowedFeeAssetIdsResponse",
```

Comment:
```suggestion
                    AllowedFeeAssetIdsResponse::name(),
```

with `use prost::Name as _;`